kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -394,15 +394,14 @@ impl<'tcx> GotocCtx<'tcx> {
                     }
                 }
             }
-            (s, ty::Tuple(substs)) => {
-                // here we have tuples of at most one length
-                if substs.len() == 1 {
-                    let overall_t = self.codegen_ty(ty);
-                    let t = substs[0];
-                    self.codegen_single_variant_single_field(s, span, overall_t, t)
-                } else {
-                    unreachable!()
-                }
+            (Scalar::Int(int), ty::Tuple(_)) => {
+                let int_u128 = int.try_to_uint(int.size()).ok().unwrap();
```

Comment:
Is this guaranteed to fit in 128 bits? If so, can we add a link to the upstream PR/code that makes this assumption?

Reply:
I've added a comment to the code.